index.js

```diff
@@ -0,0 +1,20 @@
+import express from "express";
+import axios from "axios;";
+// import bodyParser from "body-parser";
+
+const app = express();
+const port = 3000;
+
+const API_URL = "http://api.currencylayer.com/";
+const API_KEY = "0d402cae541b11d6ba5af58cc9f8585b";
```

Comment:
`API_KEYS` should never be hardcoded, I could, for example, make requests using this API Key, and depending on the pricing rules the platform has you would have to pay for it. Instead, they should be loaded using environment variables. Maybe it is a good opportunity for you to learn about it and apply such knowledge. Also, make sure to invalidate this API Key and use a different one :)